erdblick_app/app/sourcedata.panel.component.ts

```diff
@@ -233,26 +233,28 @@ export class SourceDataPanelComponent implements OnInit, AfterViewInit, OnDestro
         }
     }
 
-    selectItemWithAddress(address: bigint) {
+    selectItemWithAddress(address?: bigint) {
         let addressInRange: any;
-        if (this.addressFormat == coreLib.SourceDataAddressFormat.BIT_RANGE) {
-            const searchAddress = {
-                offset: address >> BigInt(32) & BigInt(0xFFFFFFFF),
-                size: address & BigInt(0xFFFFFFFF),
-            }
+        if (address) {
```

Comment:
Please use an early return instead of creating a large indented block :)

Reply:
You are not supposed to return here.